lib/hooks/moduleloader/index.js

```diff
@@ -109,6 +80,71 @@ module.exports = function(sails) {
           layout: path.resolve(config.appPath, 'views/layout.ejs'),
         }
       };
+
+      var hasCoffee = false, hasIced = false, hasLive = false;
+
```

Comment:
First we find out what languages are being used y walking the source files, and flagging what we find